third-party/qthread/qthread-1.10/src/interfaces/chapel/tasks-qthreads.h

```diff
@@ -411,5 +411,16 @@ c_sublocid_t chpl_task_getRequestedSubloc(void)
     return c_sublocid_any;
 }
 
+#ifdef CHPL_TASK_TASKS_BOUND_TO_PTHREADS_IMPL_DECL
+#error "CHPL_TASK_TASKS_BOUND_TO_PTHREADS_IMPL_DECL is already defined!"
+#else
+#define CHPL_TASK_TASKS_BOUND_TO_PTHREADS_IMPL_DECL 1
+#endif
+extern int chpl_qthread_tasks_bound_to_pthreads;
+static inline
+int chpl_task_tasksBoundToPthreads(void) {
+  return chpl_qthread_tasks_bound_to_pthreads;
```

Comment:
Or if you don't like that, could you change the makefile to 

```
ifeq (nemesis,$(SCHEDULER))
CFLAGS += -DTASKS_BOUND_TO_PTHREADS=1
else
CFLAGS += -DTASKS_BOUND_TO_PTHREADS=0
endif
```

that way, we're just asserting what we do know for nemesis, rather than assuming sherwood is the only scheduler that has task migration